packages/containers/tests

```diff
@@ -0,0 +1 @@
+../../test/unit/containers
```

Comment:
If it still makes sense to keep the `galaxy-containers` package around (it could be merged into `galaxy-tool-util`), I think we should keep `package/containers/tests` as a directory, with an empty `__init__.py` and the `containers` symlink pointing to `../../../test/unit/containers`

Reply:
I guess we don't. It's probably better to not release a new version (because we drop the package) than to remove almost all content :).

Reply:
Agreed that merging it into `galaxy-tool-util` seems the best option.